tests/README.md

```diff
@@ -49,11 +49,11 @@ TC_PATHS="tests/kafkatest/tests/streams/streams_upgrade_test.py::StreamsUpgradeT
 ```
 * Run tests with a specific image name
 ```
-image_name="ducker-ak-openjdk:17-buster" bash tests/docker/run_tests.sh
+image_name="ducker-ak-<JDK_IMAGE>" bash tests/docker/run_tests.sh
```

Comment:
I think @fvaleri's suggestion is to add an example for the usage of `jdk_version`